test/util.py

```diff
@@ -147,6 +154,11 @@ def fuse_test_marker():
 
 # Try to use local fusermount3
 os.environ['PATH'] = '%s:%s' % (pjoin(basename, 'util'), os.environ['PATH'])
+# Put example binaries on PATH
+os.environ['PATH'] = '%s:%s' % (pjoin(basename, 'example'), os.environ['PATH'])
```

Comment:
Hmm. Does this mean mounting through mount.fuse3 will only ever work for filesystems in $PATH?

Reply:
Note that I haven't changed this, IIUC mount.fuse has always worked like this. It eventually launches /bin/sh -c ${command}, so ${command} must be on PATH. My understanding is that when you do mount -t fuse3 ${source} /mnt/mountpoint (also if invoked via fstab), mount looks for the mount.fuse3 binary in /sbin and a couple other locations, then mount mount.fuse3 looks for ${source} on paths and executes it.